ambry-messageformat/src/main/java/com.github.ambry.messageformat/MessageFormatSend.java

```diff
@@ -86,7 +89,8 @@ private void calculateOffsets() throws IOException, MessageFormatException {
       // get size
       int messageCount = readSet.count();
       // for each message, determine the offset and size that needs to be sent based on the flag
-      infoList = new ArrayList<SendInfo>(messageCount);
+      infoList = new ArrayList<>(messageCount);
```

Comment:
can we rename this variable to sendInfoList

Reply:
Done.